### PR TITLE
feat: Added jina-embeddings-v2-base-code

### PR DESCRIPTION
**Pull request by Anush008**
## Description
As the title goes.

I've verified the embeddings produced with `sentence_transformers.SentenceTransformer`.

### CODE REVIEW

File: tests/test_text_onnx_embeddings.py [hunk 1]
```diff
@@ -47,6 +47,9 @@
     "jinaai/jina-embeddings-v2-base-de": np.array(
         [-0.0085, 0.0417, 0.0342, 0.0309, -0.0149]
     ),
+    "jinaai/jina-embeddings-v2-base-code": np.array(
+        [0.0145, -0.0164,  0.0136, -0.0170,  0.0734]
+    ),
```
Comment (joein) [inline tests/test_text_onnx_embeddings.py:52]: was it produced with:
```
from sentence_transformers import SentenceTransformer

model = SentenceTransformer('jinaai/jina-embeddings-v2-base-code', trust_remote_code=True)
s.encode(['hello world'])[:,:5]
```

(it gives me a different result: 
```
array([[ 0.20987998, -0.2358965 ,  0.19645616, -0.24518925,  1.0565469 ]],
      dtype=float32)
```

Comment (Anush008) [reply tests/test_text_onnx_embeddings.py:52]: I remember filling these values from ST for "Hello". Then after https://github.com/qdrant/fastembed/actions/runs/9962765584/job/27527279039?pr=301, I realized it was the wrong str, I got the values from there.

File: fastembed/text/pooled_normalized_embedding.py [hunk 2]
```diff
@@ -44,6 +44,14 @@
         "sources": {"hf": "jinaai/jina-embeddings-v2-base-de"},
         "model_file": "onnx/model_fp16.onnx",
```
Comment (Anush008) [inline fastembed/text/pooled_normalized_embedding.py:45]: This was unintentional. Using a quantized model with a regular name.

**Comment by Anush008** [comment]
I'm closing this for now. I'll look into the Jina models disparity.

File: tests/test_text_onnx_embeddings.py (hunk 1 above)
Comment (Anush008) [reply tests/test_text_onnx_embeddings.py:52]: `normalize_embeddings=True` made it.

**Review by joein — APPROVED** [review]

**Pull request merged**